pokerole_irl/pokerole_app/templates/index.html

```diff
@@ -7,14 +7,10 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link href="{% static 'styles.scss' %}" rel="stylesheet">
+    <link href="{% static 'styles.css' %}" rel="stylesheet">
```

Comment:
Skulle vi ikke køre med scss?

Reply:
Jeg opretter en ticket på at implementere det, men det kræver en mere omfattende installation end jeg troede. Så for nu har jeg skrevet css, så ændrer jeg det når scss er implementeret.